index.html

```diff
@@ -0,0 +1,113 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Youtube Clone</title>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+    <link href="https://fonts.googleapis.com/css?family=Roboto&display=swap" rel=“stylesheet”>
+</head>
+<body>
+    <div class="navbar">
+
+        <div class="nav-logo">
+            <i class="material-icons">menu</i>
+            <img id="logo" src="https://upload.wikimedia.org/wikipedia/commons/thumb/e/e1/Logo_of_YouTube_%282015-2017%29.svg/1280px-Logo_of_YouTube_%282015-2017%29.svg.png">
+        </div>
+
+        <div class="search-container">
+            <input class="search-input" type="text" placeholder="Search">
+            <button class="search-button" type="button"><i class="material-icons">search</i></button>
+        </div>
+
+        <div class="navbar-items">
+            <a><i class='material-icons'>apps</i></a>
+            <a><i class='material-icons'>message</i></a>
+            <a><i class='material-icons'>video_call</i></a>
+            <div class="navbar-avatar"></div>
+        </div>
+    </div>
+    <div class="main">
```

Comment:
[OPTIONAL]
It's best practice to use the `main` tag for the main content of your page. The main content of your page it's the most important part of it. In this project's case, the video section